### PR TITLE
WIP New multitask

### PR DESCRIPTION
**Pull request by dimakarp1996**

### CODE REVIEW

File: skills/dff_gossip_skill/dialogflows/flows/gossip.py [hunk 1]
```diff
@@ -314,21 +314,21 @@ def get_random_judgement_for_emotion(emotion):
...
 
 
-supported_cobot_topics = [
-    "Entertainment_Movies",
-    "Entertainment_Music",
-    "Entertainment_Books",
+supported_topics = [
+    "Movies_TV",
+    "Music",
+    "Books&Literature",
```
Comment (dilyararimovna) [inline skills/dff_gossip_skill/dialogflows/flows/gossip.py:320]: а почему ты старые удаляешь? код должен быть совместим. Здесь должны быть и старые, и новые

**Review by dilyararimovna — CHANGES_REQUESTED** [review]

File: skills/dff_gossip_skill/dialogflows/flows/gossip.py (hunk 1 above)
Comment (dimakarp1996) [reply skills/dff_gossip_skill/dialogflows/flows/gossip.py:358]: ok

File: dev_requirements.txt [hunk 2]
```diff
@@ -2,4 +2,4 @@ sphinx==2.1.2
 recommonmark==0.5.0
 sphinx_rtd_theme
 Pygments==2.4.2
-flake8
\ No newline at end of file
+flake8
```
[the remark this answers is not in the document]
Comment (dimakarp1996) [reply dev_requirements.txt]: ok

File: common/utils.py [hunk 3]
```diff
@@ -752,13 +807,22 @@ def get_topics(annotated_utterance, probs=False, default_probs=None, default_lab
...
     if which == "all":
-        answer_labels = cobot_topics_labels + cobot_da_topics_labels
-        answer_probs = {**cobot_topics_probs, **cobot_da_topics_probs}
+        answer_labels = cobot_topics_labels + cobot_da_topics_labels + topics_labels
+        answer_probs = {**cobot_topics_probs, **cobot_da_topics_probs, **topics_probs}
     elif which == "cobot_topics":
         answer_probs, answer_labels = cobot_topics_probs, cobot_topics_labels
     elif which == "cobot_dialogact_topics":
         answer_probs, answer_labels = cobot_da_topics_probs, cobot_da_topics_labels
+    elif which == "topics":
```
[the remark this answers is not in the document]
Comment (dimakarp1996) [reply common/utils.py]: ok

File: annotators/combined_classification/combined_classifier.json [hunk 4]
```diff
@@ -18,165 +18,39 @@
...
+            "backbone_model":"{BACKBONE}",
+            "save_path":"{MODELS_PATH}/model",
+            "load_path":"{MODELS_PATH}/model",
+            "tasks":{
+               "emo":{"type":"classification","options":7, "multilabel":true},
+               "sentiment":{"type":"classification","options":3},
+               "toxic":{"type":"classification","options":8, "multilabel": true},
+               "factoid":{"type":"classification","options":2},
+               "midas":{"type":"classification","options":15},
+               "topics":{"type":"classification","options":33}
```
[the remark this answers is not in the document]
Comment (dimakarp1996) [reply annotators/combined_classification/combined_classifier.json]: ok

File: annotators/combined_classification/README.md [hunk 5]
```diff
@@ -1 +1 @@
-BERT Base model for 6 tasks - cobot topics cobot dialogact topics cobot dialogact intent emotion sentiment toxic
+DistilBERT Base Uncased model for 6 tasks - emotion sentiment toxic factoid midas topic classification
```
[the remark this answers is not in the document]
Comment (dimakarp1996) [reply annotators/combined_classification/README.md]: ok

File: annotators/combined_classification/Dockerfile [hunk 6]
```diff
@@ -1,14 +1,8 @@
-FROM deeppavlov/base-gpu:0.12.1
-RUN pip install git+https://github.com/deeppavlov/DeepPavlov.git@0.12.1
+FROM deeppavlov/base-gpu:0.17.5
+RUN pip install git+https://github.com/dimakarp1996/DeepPavlov.git@clean_multitask_ready
```
[the remark this answers is not in the document]
Comment (dimakarp1996) [reply annotators/combined_classification/Dockerfile]: ok

**Comment by dimakarp1996** [comment]
The reasons of the inability to merge this PR were stated in the dp_dream channel on Friday, 7 October. So, instead, I have made a New PR with the 9in1 model.

**Pull request closed without merging**